### DISCUSSION TITLE
ESP32 maximum resolution?

### DISCUSSION THREAD

**Discussion by radontec** [Bodmer/TFT_eSPI#1672] [Compatible displays and setup files]
Hello all,

I have a question of interest regarding displays and the ESP32.  
The largest screens used on an ESP32 are 3.5inch with 480x320 
I have not seen one with a higher resolution, do you know why? Or is there some kind of limit from the processor power or number of GPIOs which doesn't allow a higher resolution?

**Comment by Bodmer** [comment]
The software potentially allows 32000 x 32000 pixels!

The largest display I have used is an SSD1963 based TFT which is 800 x 480.

https://www.buydisplay.com/7-tft-screen-touch-lcd-display-module-w-ssd1963-controller-board-mcu

**Reply by BrotherCreamy** [reply]
Hi Bodmer,

Do you know of any SPI display driver ICs that would support a similar resolution, that are supported by the library?

Thanks,
Brother

**Comment by bionicbone** [comment]
@Bodmer I don't suppose you recall which parallel interface you used ?  6800 or 8080

![image](https://user-images.githubusercontent.com/7845867/166411186-afc53fa1-3d58-4c5a-8d99-43f399039e2c.png)

EDIT: It looks like the board ships configured but with excellent soldering skills you could change this...

![image](https://user-images.githubusercontent.com/7845867/166413386-09d62171-eb0a-4094-a5e9-348127da28b9.png)


And font chip?

![image](https://user-images.githubusercontent.com/7845867/166411337-734e9dcb-c735-4036-90cf-d09ac3fa5b0f.png)

EDIT: I am guessing the font chip is not used. However, researching I'd probably plump for ER3304 with 14 ASCII fonts capability.

**Reply by Bodmer** [reply]
8080 is the correct one.
The font chip is not supported by the library, you would need to write functions to read it and render the characters. Just leave that option not selected.

**Comment by radontec** [comment]
Why did you send this to RadonTec? Please explain.


Claudia



Falls Sie mit unserem Service zufrieden waren, würden wir uns über eine positive Bewertung freuen.
Es dauert nur wenige Sekunden :) Danke --> Hier bewerten<https://g.page/r/CQoZX_sKi3TYEAg/review>

***@***.***

RadonTec GmbH
Hauptstr. 5
89426 Wittislingen | Deutschland
Tel: (+49) 09076 - 919 98 35
Ust.ID: DE310274795
Finanzamt Dillingen a. d. Donau
https://radontec.de<https://radontec.de/>
--------------------------------------------------------
***@***.***

Seit 2019 ist RadonTec GmbH ein staatlich anerkannter
Anbieter von Radonmessungen für Arbeitsplätze, gemäß
§155 Strahlenschutzverodnung(BfS).

Von: Kevin Guest ***@***.***>
Gesendet: Dienstag, 3. Mai 2022 08:13
An: Bodmer/TFT_eSPI ***@***.***>
Cc: Support ***@***.***>; Author ***@***.***>
Betreff: Re: [Bodmer/TFT_eSPI] ESP32 maximum resolution? (Discussion #1672)

Sie erhalten nicht oft eine E-Mail von ***@***.*** Erfahren Sie, warum dies wichtig ist<https://aka.ms/LearnAboutSenderIdentification>

@Bodmer<https://github.com/Bodmer> I don't suppose you recall which parallel interface you used ? 6800 or 8080

[Das Bild wurde vom Absender entfernt. image]<https://user-images.githubusercontent.com/7845867/166411186-afc53fa1-3d58-4c5a-8d99-43f399039e2c.png>

**Reply by Bodmer** [reply]
@radontec started the discussion on this repository, so you get notification for posting on this thread. Switch off notifications if you do not want them!
